$MirOS: ports/devel/cvs/patches/patch-src_client_c,v 1.3 2010/09/18 22:35:09 tg Exp $
--- src/client.c.orig	Sun Oct  2 15:16:58 2005
+++ src/client.c	Sat Jun 11 03:06:20 2011
@@ -20,6 +20,8 @@
 #include "buffer.h"
 #include "save-cwd.h"
 
+__RCSID("$MirOS: ports/devel/cvs/patches/patch-src_client_c,v 1.3 2010/09/18 22:35:09 tg Exp $");
+
 #ifdef CLIENT_SUPPORT
 
 # include "log-buffer.h"
@@ -3901,8 +3903,12 @@ open_connection_to_server (cvsroot_t *ro
 	case ext_method:
 #ifdef NO_EXT_METHOD
 	    error (0, 0, ":ext: method not supported by this port of CVS");
+	    if (0)
+	case extssh_method:
+		error (0, 0, ":extssh: method not supported by this port of CVS");
 	    error (1, 0, "try :server: instead");
 #else /* ! NO_EXT_METHOD */
+	case extssh_method:
 	    start_rsh_server (root, to_server_p,
                               from_server_p);
 #endif /* NO_EXT_METHOD */
@@ -4362,7 +4368,7 @@ send_modified (const char *file, const c
 	  sprintf (tmp, "%lu\n", (unsigned long) newsize);
 	  send_to_server (tmp, 0);
 
-          send_to_server (buf, newsize);
+          send_to_server ((const void *)buf, newsize);
         }
     }
     else
@@ -4415,7 +4421,7 @@ send_modified (const char *file, const c
 	 * one.
 	 */
 	if (newsize > 0)
-	    send_to_server (buf, newsize);
+	    send_to_server ((const void *)buf, newsize);
     }
     free (buf);
     free (mode_string);
